pkg/subscription/handler.go

```diff
@@ -269,7 +269,15 @@ func (h *Handler) startSubscription(ctx context.Context, id string, executor Exe
 
 // executeSubscription will keep execution the subscription until it ends.
 func (h *Handler) executeSubscription(buf *graphql.EngineResultWriter, id string, executor Executor) {
-	// err := executor.Execute(ctx, node, buf)
+	//err := executor.Execute(ctx, node, buf)
```

Comment:
I forgot to remove this comment when I refactored this file - maybe we can remove it now?